docs/wiki/deployment/yara.md

```diff
@@ -0,0 +1,138 @@
+# Introduction
+
+I recently put [YARA](https://github.com/plusvic/yara) inside [osquery](https://osquery.io) and thought I would provide some details on how to use it. There are two YARA related tables in osquery, which serve very different purposes. The first table, called **yara_events**, uses osquery's pub-sub framework to monitor for filesystem changes and will execute YARA when a file change event fires. The second table, called **yara**, is an on-demand YARA scanning table.
+
+# Configuration
+
+The configuration for osquery is simple. Here is an example config (I've left out the non-relevant parts):
+
+```
+  // Description of the YARA feature.
+  "yara": {
+    "signatures": {
+      // Each key is an arbitrary group name to give the signatures listed
+      "sig_group_1": [ "/Users/wxs/sigs/foo.sig", "/Users/wxs/sigs/bar.sig" ],
```

Comment:
Right now the YARA table will check in ```/var/osquery``` if the sigfile is not an absolute path. I can change it to be ```/etc/osquery/yara/``` if you would like? I only went with ```/var/osquery``` because it was mentioned in an earlier PR. I will also make the config parser have the same logic.